Src/Utils/Paths.ts

```diff
@@ -1,6 +1,14 @@
+import finder from "find-package-json";
 import path from "path";
 
-export const ROOT_DIR: string = path.normalize(`${__dirname}/../../`);
+// crawl up parent directories until we find the package.json of the calling software
+let lRootDir: string = process.cwd();
+for (const lPackageJson of finder())
```

Comment:
```suggestion
for (const lPackageJson of find())
```

to keep name same as declaration export

Reply:
I'm stuck away from my computer disputing some fraudulent charges. Can you implement these changes and merge them in?